mpisppy/cylinders/lagranger_bounder.py

```diff
@@ -113,4 +113,7 @@ def finalize(self):
         '''
         self.final_bound = self._update_weights_and_solve(self.A_iter)
         self.bound = self.final_bound
+        if self.opt.extensions is not None and \
+            hasattr(self.opt.extobject, 'post_everything'):
```

Comment:
Ben's suggested that we *not* do conditional exceptions and instead add a pass to the base class for extensions when we add a new extension callout.

Reply:
Oh I ended up doing both. I'll remove the conditional exceptions